robot-server/tests/runs/router/test_actions_router.py

```diff
@@ -248,28 +204,32 @@ async def test_create_run_action_not_current(
         is_current=False,
     )
 
-    decoy.when(run_store.get(run_id="run-id")).then_return(prev_run)
+    decoy.when(mock_run_store.get(run_id="run-id")).then_return(prev_run)
 
     with pytest.raises(ApiError) as exc_info:
         await create_run_action(
             runId="run-id",
             request_body=RequestModel(
                 data=RunActionCreate(actionType=RunActionType.PLAY)
             ),
-            run_store=run_store,
+            run_store=mock_run_store,
         )
 
     assert exc_info.value.status_code == 409
     assert exc_info.value.content["errors"][0]["id"] == "RunStopped"
-
-    decoy.verify(run_store.upsert(run=matchers.Anything()), times=0)
+    # TODO (tz): does this makes sense?
+    decoy.verify(
+        mock_run_store.insert_action(
+            run_id=prev_run.run_id, action=matchers.Anything()
+        ),
+        times=0,
+    )
```

Comment:
This makes sense to me; I think you can remove this `# TODO`.

If you want, you could also make it like this:

```suggestion
    decoy.verify(
        mock_run_store.insert_action(
            run_id=matchers.Anything(), action=matchers.Anything()
        ),
        times=0,
    )
```